examples/lighting-app/nrfconnect/CMakeLists.txt

```
@@ -30,16 +30,32 @@ if (EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/boards/${BOARD}.conf)
   list(APPEND CONF_FILE boards/${BOARD}.conf)
 endif()
 
-option(BUILD_WITH_DFU "Build target with Device Firmware Upgrade support" OFF)
-if(BUILD_WITH_DFU)
+set(BUILD_WITH_DFU "MATTER" CACHE STRING "Build target with Device Firmware Upgrade support")
+if(NOT ${BUILD_WITH_DFU} STREQUAL "OFF")
+    if(${BUILD_WITH_DFU} STREQUAL "BLE")
+        message(BLE)
+        if(${BOARD} STREQUAL "nrf5340dk_nrf5340_cpuapp")
```

nit:
In CMake you don't need to use ${} syntax when comparing variables:

```suggestion
        if(BOARD STREQUAL "nrf5340dk_nrf5340_cpuapp")
```